app/assets/stylesheets/markets.scss

```diff
@@ -0,0 +1,99 @@
+// Place all the styles related to the markets controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+.details {
+  text-align: left;
+  margin: 0 auto;
+}
+.details tr {
+  margin: 10px 0;
+}
+.details td:first-child {
+  text-align: right;
+  border-right: 1px solid #ccc;
+  padding-right: 3px;
+  color: #9a9a9a;
+}
+.details td:last-child {
+  padding-left: 10px;
+}
+
+.market-vendors a {
+  padding-left: 5px;
+}
+
+.market-vendors div {
+  padding: 5px;
+  margin-bottom: 20px;
+  border-radius: 3px;
+  display: inline;
+  color: #9a9a9a;
+}
+.market-vendors ul {
+  list-style-type: none;
+  padding: 0;
+}
+.market-vendors li {
+  border-bottom: 1px solid #ccc;
+  padding: 5px;
+}
+
+/* EDIT FORM */
+.edit-form {
+  width: 50%;
+  background-color: lighten(#ccc, 10);
+  padding: 5px;
+  border-radius: 3px;
+  margin: 0 auto;
+}
+.edit-form form {
+  width: 50%;
+  margin: 20px auto;
+}
+.edit-form label,
+.edit-form input {
+  margin: 0 0 5px 0;
```

Comment:
Maybe next time you can try to set up your CSS so you don't have to override it like you needed to here?